Objects/unicodeobject.c

```diff
@@ -205,8 +205,7 @@ unicode_decode_utf8(const char *s, Py_ssize_t size,
 static int
 unicode_decode_utf8_writer(_PyUnicodeWriter *writer,
                            const char *s, Py_ssize_t size,
-                           _Py_error_handler error_handler, const char *errors,
-                           Py_ssize_t *consumed);
```

Comment:
This is needed for my next PR.

Reply:
Oh, interesting :-) I'm now curious about your next PR! I modified my PR to restore this feature.

Reply:
See #120365.